float64     target_heading
float64     distance
int8        waypoint
bool        stop_robot
bool        done
